### PR TITLE
feat(api-client): Identity Provider API

### PR DESCRIPTION
**Pull request by Yserz**

### CODE REVIEW

File: packages/api-client/src/main/team/identityprovider/IdentityProvider.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+interface IdentityProvider {
```
Comment (ffflorian) [inline packages/api-client/src/main/team/identityprovider/IdentityProvider.ts]: Add license header

File: packages/api-client/src/main/team/identityprovider/IdentityProvider.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  id: string;
+  metadata: IdentityProviderMetaData;
+  extraInfo: string;
+}
+
+interface IdentityProviderMetaData {
+  issuer: string;
+  requestURI: string;
+  certAuthnResponse: string[];
```
Comment (ffflorian) [inline packages/api-client/src/main/team/identityprovider/IdentityProvider.ts]: Is this a typo? Also: sorting :wink:

File: packages/api-client/src/main/team/identityprovider/IdentityProviders.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import {IdentityProvider} from './IdentityProvider';
```
Comment (ffflorian) [inline packages/api-client/src/main/team/identityprovider/IdentityProviders.ts]: Add license header

File: packages/api-client/src/main/team/identityprovider/IdentityProvider.ts (hunk 2 above)
Comment (Yserz) [reply packages/api-client/src/main/team/identityprovider/IdentityProvider.ts]: Nah, that's how the backend calls it 😉

**Pull request merged**